scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/sop.yaml

```diff
@@ -199,6 +199,34 @@ SOP-8_6.62x9.15mm_P2.54mm:
   num_pins_x: 0
   num_pins_y: 4
 
+SOP-10_3.9x4.9mm_P1mm:
+  size_source: 'https://www.holtek.com/documents/10179/11842/HT42B534-xv110.pdf'
+  body_size_x:
+    minimum: 3.80
+    maximum: 4.00
+  body_size_y:
+    minimum: 4.80
+    maximum: 5.00
```

Comment:
Where did you get the min-max values from?

Reply:
Sorry, I didn't realize you were able to just write exact values! These don't have tolerances, so I changed them.